### ISSUE TITLE
Detecting keydown events while typing in the input field with WebGLInput

### ISSUE THREAD

**Issue by nicanorrr** [kou-yeung/WebGLInput#40]
Hi, while I'm typing in the InputField, I tried to detect keydown events using Input.anyKeyDown so that after a key is typed into the input field, the string is saved through playerprefs. It works in the editor but isn't working when I build to WebGL. Am I correct in assuming it is because of how WebGLInput works that it blocks some of the keyboard input while a InputField is being typed into?

How I'm doing it is something like this:

    void Update(){
        if (Input.anyKeyDown){
            // my code for saving data
        }
    }

If yes, is there another way I can allow my unity code to detect when a player has typed into the input field? Thank you :D

**Comment by kou-yeung** [comment]
WebGLInput plugins need keyboard input to work

FYR: Keyboard input and focus handling
https://docs.unity3d.com/2018.4/Documentation/Manual/webgl-input.html

**Comment by kou-yeung** [comment]
how about it?
```
void Update() {
    if(Input.anyKeyDown) SavingData();
}
// add value change event to input field
void OnValueChange() {
#if UNITY_WEBGL && !UNITY_EDITOR
    SavingData();
#endif
} 
void SavingData(){
}
```

**Comment by nicanorrr** [comment]
When I tested it, Input.anyKeyDown is always false while I'm typing in the input field with WebGLInput. I'm not sure why.

However your solution pointed me in the right direction :) OnValueChanged didn't work in script but I was able to set the function in the Inspector. This works in WebGL.

![image](https://user-images.githubusercontent.com/65482727/82538166-e2697200-9b7d-11ea-973a-6d003a7310b8.png)
